lib/prefix.c

```diff
@@ -966,18 +966,12 @@ void masklen2ip(const int masklen, struct in_addr *netmask)
 }
 
 /* Convert IP address's netmask into integer. We assume netmask is
-   sequential one. Argument netmask should be network byte order. */
+ * sequential one. Argument netmask should be network byte order. */
 uint8_t ip_masklen(struct in_addr netmask)
 {
 	uint32_t tmp = ~ntohl(netmask.s_addr);
-	if (tmp)
-		/* clz: count leading zeroes. sadly, the behaviour of this
-		 * builtin
-		 * is undefined for a 0 argument, even though most CPUs give 32
-		 */
-		return __builtin_clz(tmp);
-	else
-		return 32;
+
+	return tmp ? __builtin_clz(tmp) : 32;
```

Comment:
Please restore the comment, this is non-obvious behaviour.

Reply:
Ack